routes/secretWord.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+
+
+router.get('/', (req, res) => {
+  if (!req.session.secretWord) {
+    req.session.secretWord = 'syzygy';
+  }
+
+  res.render('secretWord', { secretWord: req.session.secretWord });
+});
+
+router.post('/', (req, res) => {
+  if (req.body.secretWord.toUpperCase()[0] == 'P') {
```

Comment:
** this isn't required but just some insight based on my experience! ** 

usually it'd be better to create a validation function that will return a `true/false` value, and then you can execute the `if/else` block afterwards based on that value.  for example:

``` 
const validate = () => {
   if (req.body.secretWord.toUppercase()[0] === 'P') {
return false }

return true
} 
```

and then in this route, you could have: 

```
 const isValid = validate(req.body.secretWord)

if (!isValid) {
req.flash('error', "That word won't work!");
    req.flash('error', "You can't use words that start with p.");
} else {
    req.session.secretWord = req.body.secretWord;
    req.flash('info', 'The secret word was changed.');
  }
```


this helps in the event where you want to check for multiple validation conditions, and it also makes the validation logic reusable in the future

Reply:
Love this insight! I'll keep this in mind for the final project as well. Changes made, thank you @jcheng1022